.changeset/@graphql-mesh_plugin-opentelemetry-532-dependencies.md

```diff
@@ -0,0 +1,8 @@
+---
+'@graphql-mesh/plugin-opentelemetry': patch
+---
+
+dependencies updates: 
+
+- Updated dependency [`@whatwg-node/promise-helpers@1.3.0-alpha-20250309193428-c964a818386b30d5ef00968233f6f0e15674fb8e` ↗︎](https://www.npmjs.com/package/@whatwg-node/promise-helpers/v/1.3.0) (from `^1.0.0`, in `dependencies`)
```

Comment:
Instead of changing `promise-helpers` in one place, I'd use `resolutions` to make sure the new version of promise helpers works everywhere else.

Reply:
It is also in the `resolution` yes :-)

Reply:
Oh I missed it then :facepalm: